riseagain47/EPIC-Mail
# Chore(server) setup database integration [Delivers #164725004]

#### What does this PR do?
This PR integrates Unit and Integration Testing to the Application using Mocha and Chai

#### Description of Task to be completed?
Have the following:
- An .env file for database URL
- A connectDB.js file for database migration.
- A db folder containing db.js files for creating tables upon database migration.

#### How should this be manually tested?
- This server migration to ElephantSql should be done by running `npm run create-table`
- On success with table creation, a user should see a message on the terminal `All tables created successfully`

#### Any background context you want to provide?
User should make sure the computer is connected to the internet before performing the database actions.

#### What are the relevant pivotal tracker stories?
Chore(server) #164725004 - setup database integratio
